Latest flake8 fails

The latest version of flake8 fails for `ratbagd.py`
```
ratbagd.py:196:33: F523 '...'.format(...) has unused arguments at position(s): 0
```
This is the relevant function:
https://github.com/libratbag/libratbag/blob/ceaa3c3ad5c7b4b7ebe1bc5ca8ae1c1a30ecb006/tools/ratbagd.py#L187-L203

/cc @FFY00 can you add flake8 to the arch CI in Piper and libratbag? Then we would have noticed it earlier, currently it only is used in Piper on Fedora and Ubuntu.

--- a/tools/ratbagd.py
+++ b/tools/ratbagd.py
@@ -193,7 +193,7 @@ class _RatbagdDBus(GObject.GObject):
         # args to .Set are "interface name", "function name",  value-variant
         val = GLib.Variant("{}".format(type), value)
         if readwrite:
-            pval = GLib.Variant("(ssv)".format(type), (self._interface, property, val))
+            pval = GLib.Variant("(ssv)", (self._interface, property, val))
             self._proxy.call_sync("org.freedesktop.DBus.Properties.Set",
                                   pval, Gio.DBusCallFlags.NO_AUTO_START,
                                   2000, None)

